deploy/awstools/afitools.py

```diff
@@ -1,5 +1,7 @@
 """ Tools to help manage afis. """
 
+from __future__ import annotations
```

Comment:
Thank you for doing this across all files.  Since it changes the fundamental behavior of type annotations, it is good to be consistent in how we're using it.  It would be nice if mypy had a check (or something somewhere) that makes sure that we continue to have the import in all python files.